vscode-debugger/README.md

```diff
@@ -0,0 +1,12 @@
+# VS Code Debugger
+
+Use our [VS Code plugin](https://marketplace.visualstudio.com/items?itemName=temporal-technologies.temporalio) to debug a running or completed Workflow Execution. See this blog post for a walkthrough:
```

Comment:
Wanna record a video too?

Reply:
Can just be screen capture, but I guess that might go on the extension README.

Reply:
Will link video from post